diff --git a/cmake/onigmo.cmake b/cmake/onigmo.cmake
index 7d7b6c55..c400c349 100644
--- a/cmake/onigmo.cmake
+++ b/cmake/onigmo.cmake
@@ -36,7 +36,7 @@ ExternalProject_Add(onigmo
   SOURCE_DIR ${ONIGMO_SRC}
   INSTALL_DIR ${ONIGMO_DEST}
   CONFIGURE_COMMAND ./configure ${AUTOCONF_HOST_OPT} --with-pic --disable-shared --enable-static --prefix=${ONIGMO_DEST}
-  CFLAGS=-std=gnu99\ -Wall\ -pipe\ -g3\ -O3\ -funroll-loops
+  CFLAGS=-std=gnu99\ -Wall\ -pipe\ -g3\ -O3\ -funroll-loops\ -fPIC
   BUILD_COMMAND $(MAKE)
   INSTALL_COMMAND $(MAKE) DESTDIR= install)
 endif()
-- 
2.31.1

