From 0165905ad5356c0052e79b14b1f9a77cdfbaa19b Mon Sep 17 00:00:00 2001
From: Benjamin Kircher <bkircher@0xadd.de>
Date: Wed, 21 Apr 2021 10:13:29 +0200
Subject: [PATCH 2/3] CMake: fix up install paths


diff --git a/CMakeLists.txt b/CMakeLists.txt
index 33e909b6..9c225122 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -797,9 +797,9 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
   set(FLB_INSTALL_INCLUDEDIR "include")
 else()
   set(FLB_INSTALL_BINDIR ${CMAKE_INSTALL_FULL_BINDIR})
-  set(FLB_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}/${FLB_OUT_NAME}")
+  set(FLB_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}")
   set(FLB_INSTALL_CONFDIR "${CMAKE_INSTALL_SYSCONFDIR}/${FLB_OUT_NAME}/")
-  set(FLB_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_PREFIX}/include")
+  set(FLB_INSTALL_INCLUDEDIR /usr/include)
 endif()
 
 # Instruct CMake to build the Fluent Bit Core
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index fb3ed782..79d8ac71 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -445,7 +445,7 @@ if(FLB_BINARY)
       "${PROJECT_SOURCE_DIR}/init/systemd.in"
       ${FLB_SYSTEMD_SCRIPT}
       )
-    install(FILES ${FLB_SYSTEMD_SCRIPT} COMPONENT binary DESTINATION /lib/systemd/system)
+    install(FILES ${FLB_SYSTEMD_SCRIPT} COMPONENT binary DESTINATION /usr/lib/systemd/system)
     install(DIRECTORY DESTINATION ${FLB_INSTALL_CONFDIR} COMPONENT binary)
   elseif(IS_DIRECTORY /usr/share/upstart)
     set(FLB_UPSTART_SCRIPT "${PROJECT_SOURCE_DIR}/init/${FLB_OUT_NAME}.conf")
-- 
2.33.0

