From e3be53c5e9cecfd1d5024ba44069910320411793 Mon Sep 17 00:00:00 2001
From: Takahiro Yamashita <nokute78@gmail.com>
Date: Sun, 7 Nov 2021 08:10:36 +0900
Subject: [PATCH 4/5] tests: runtime: in_proc: modify absent process
 name(#4274)

Signed-off-by: Takahiro Yamashita <nokute78@gmail.com>

diff --git a/tests/runtime/in_simple_systems.c b/tests/runtime/in_simple_systems.c
index 00a0e949b..3778584bd 100644
--- a/tests/runtime/in_simple_systems.c
+++ b/tests/runtime/in_simple_systems.c
@@ -191,7 +191,7 @@ void flb_test_in_proc_absent_process(void)
     in_ffd = flb_input(ctx, (char *) "proc", NULL);
     TEST_CHECK(in_ffd >= 0);
     flb_input_set(ctx, in_ffd, "tag", "test",
-                  "interval_sec", "1", "proc_name", "",
+                  "interval_sec", "1", "proc_name", "-",
                   "alert", "true", "mem", "on", "fd", "on", NULL);
 
     out_ffd = flb_output(ctx, (char *) "lib", &cb);
-- 
2.34.1

